pkg/kubelet/kubelet.go

```diff
@@ -757,6 +763,16 @@ type Kubelet struct {
 
 	// handlers called during the tryUpdateNodeStatus cycle
 	setNodeStatusFuncs []func(*api.Node) error
+
+	// TODO: think about moving this to be centralized in PodWorkers in follow-on.
```

Comment:
Question: Who is expected to make pod admission decisions? Should it be pod workers or should be done by kubelet object itself, before spinning up a pod worker?